pyblish_lite/model.py

```diff
@@ -629,16 +630,17 @@ def filterAcceptsRow(self, source_row, source_parent):
         """Exclude items in `self.excludes`"""
         model = self.sourceModel()
         item = model.items[source_row]
-
         key = getattr(item, "filter", None)
         if key is not None:
             regex = self.filterRegExp()
             if regex.pattern():
                 match = regex.indexIn(key)
                 return False if match == -1 else True
-
         for role, values in self.includes.items():
             data = getattr(item, role, None)
+            if hasattr(data, '__iter__'):
+                return True if any(value in values for value in data) else False
```

Comment:
What about that:
```
        for role, values in self.includes.items():
            includes_list = [([x] if isinstance(x, (list, tuple)) else x) for x in getattr(item, role, None)]
            return any(include in values for include in includes_list)
```

As about calling this method, from my experiments it occurs that it is not called as often: it is called only when an item on the list has changed, so in fact during the execution of plugins, it is not called during scrolling of the list etc.

My experiments ware based on inserting
`
print "I'm filtering"
`
statement` in this function :) But the results should be correct